table/server/underdb/glue/src/main/java/alluxio/table/under/glue/Property.java

```diff
@@ -208,5 +244,11 @@ public String getDefaultValue() {
     public static final String CATALOG_ID = "aws.catalog.id";
     public static final String AWS_GLUE_ACCESS_KEY = "aws.accesskey";
     public static final String AWS_GLUE_SECRET_KEY = "aws.secretkey";
+    public static final String AWS_ENABLE_PROXY = "aws.enable.proxy";
```

Comment:
Do we need this? Instead, can we assume that if `aws.proxy.host` is set, then proxy is enabled? We would be able to do this if the proxy is enabled IFF `aws.proxy.host` is set.

Otherwise, if we need to keep this, can this be renamed to `aws.proxy.enable`? It would feel more consistent with the other proxy related properties.

Lastly, is this an aws type of configuration, or a glue specific configuration? If it is glue specific, I feel like we don't even need the `aws.` prefix, since this is for the glue UDB?

Reply:
You are right, I changed to use `aws.proxy.host` as trigger. These configurations are used to building the AWS client configurations, it may better to maintain the `aws.` prefix?